packages/gatsby-plugin-schema-snapshot/gatsby-node.js

```diff
@@ -0,0 +1,34 @@
+const fs = require(`fs`)
+const path = require(`path`)
+
+exports.createSchemaCustomization = ({ actions, reporter }, options = {}) => {
+  const { createTypes, printTypeDefinitions } = actions
+
+  if (!printTypeDefinitions) {
+    reporter.error(
+      `\`gatsby-plugin-schema-snapshot\` needs Gatsby v2.13.55 or above.`
+    )
```

Comment:
```suggestion
    )
    return
```
seems like early bail make sense here given we log problem and we know rest won't work?

Reply:
oops yes sure